presto-docs/src/main/sphinx/release/release-347.md

```diff
@@ -0,0 +1,62 @@
+# Release 347 (25 Nov 2020)
+
+## General Changes
+
+* Add `ALTER VIEW ... SET AUTHORIZATION` syntax for changing owner of the view. ({issue}`5789`)
+* Add support for `INTERSECT ALL` and `EXCEPT ALL`. ({issue}`2152`)
+* Add {func}`contains_sequence` function. ({issue}`5593`)
+* Support defining cluster topology (used for query scheduling) using network subnets. ({issue}`4862`) 
+* Improve query performance by reducing worker to worker communication overhead. ({issue}`5905`, {issue}`5949`)
+* Allow disabling client HTTP response compression, which can improve throughput over fast network links.
+  Compression can be disabled globally via the `query-results.compression-enabled` config property, for CLI via 
+  the `--disable-compression` flag, and for the JDBC driver via the `disableCompression` driver property. ({issue}`5818`)
+* Rename ``rewrite-filtering-semi-join-to-inner-join`` session property to ``rewrite_filtering_semi_join_to_inner_join``. ({issue}`5954`)
+* Throw a user error when session property value cannot be decoded. ({issue}`5731`)
+* Fix query failure when expressions that produce values of type `row` are used in a `VALUES` clause. ({issue}`3398`)
+
+## Security Changes
+
+* Add support for multiple LDAP bind patterns. ({issue}`5874`)
+* Include groups for view owner when checking permissions for views. ({issue}`5945`)
+
+## JDBC Driver Changes
+
+* Implement `addBatch()`, `clearBatch()` and `executeBatch()` methods in `PreparedStatement`. ({issue}`5507`)
+
+## CLI Changes
+
+* Add support for providing queries to presto-cli via shell redirection. ({issue}`5881`)
+
+## Docker Image Changes
+
+* Update Presto docker image to use Centos 8 as the base image. ({issue}`5920`)
+
+## Hive Connector Changes
+
+* Add support for `ALTER VIEW  ... SET AUTHORIZATION` SQL syntax to change the view owner. This supports Presto and Hive views. ({issue}`5789`)
+* Allow configuring HDFS replication factor via the `hive.dfs.replication` config property. ({issue}`1829`)
+* Add access checks for tables in Hive Procedures. ({issue}`1489`)
+* Decrease latency of `INSERT` and `CREATE TABLE AS ...` queries by updating table and column statistics in parallel. ({issue}`3638`)
+* Fix leaking S3 connections when querying Avro tables. ({issue}`5562`)
+
+## Kudu Connector Changes
+
+* Add dynamic filtering support for Kudu. It can be enabled using ``kudu.dynamic-filtering.wait-timeout``
```

Comment:
Optional suggestion: `Add dynamic filtering support` should be enough as this is already in Kudu section. 

The property seems to be a `Duration`, and the feature is enabled by "setting a non-zero value" for the property. Current sentence structure makes it sound like it's a boolean to me.

```
Add dynamic filtering support. It can be enabled by setting a non-zero duration value for ``kudu.dynamic-filtering.wait-timeout`` config property or ``dynamic_filtering_wait_timeout`` session property.
```